appinventor/components/src/com/google/appinventor/components/common/YaVersion.java

```diff
@@ -680,11 +680,12 @@ private YaVersion() {
 
   public static final int GYROSCOPESENSOR_COMPONENT_VERSION = 1;
 
-  // For HORIZONTALARRANGEMENT_COMPONENT_VERSION 2:
+  // For HORIZONTALARRANGEMENT_COMPONENT_VERSION 3:
+  // - The scrollable property was added
```

Comment:
The comments are not quite right.   AlignHorizontal and AlignVertical and color (I think) were part of the upgrade to version 3.   Then scrollabe is part of the upgrade to version 4.
One reason you got confused is that the previous version of YaVersion had a bug.   See my email to you and to Jeff Schiller.